src/main/java/gov/usgs/owi/nldi/swagger/model/Feature.java

```diff
@@ -0,0 +1,13 @@
+package gov.usgs.owi.nldi.swagger.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Feature {
+	@JsonProperty("name")
+	public String name;
+	@JsonProperty("uri")
+	public String uri;
+	@JsonProperty("comid")
+	public String comid;
+
+}
```

Comment:
I like to see an empty line at the end of files. Github flagged this too. The .xml files modified are also flagged (left over from other people)